bears/rest/RSTcheckBear.py

```diff
@@ -32,16 +44,39 @@ class RSTcheckBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         code_block_language_ignore: list=()):
+                         code_block_language_ignore: list=(),
+                         ignore_directives: list=(),
+                         ignore_roles: list=()):
         """
         :param code_block_language_ignore:
             Comma seperated list for languages of which code blocks
             should be ignored. Code block of following languages
             can be detected: ``bash``, ``c``, ``cpp``, ``json``,
             ``python``, ``rst``.
+
+        :param ignore_directives:
+             Comma sepereated list of directives to be ignored
+
+        :param ignore_roles:
+             Comma seperated list of roles to be ignored
         """
+
         args = ()
+
         if code_block_language_ignore:
             args = ('--ignore-language=' +
                     ','.join(code_block_language_ignore),)
-        return args + (filename,)
+
+        args += ((
+            '--ignore-directives=' + ','
+            .join(ignore_directives),)
+            if ignore_directives else ()
+        )
+
+        args += ((
+            '--ignore-roles=' + ','
+            .join(ignore_roles),)
+            if ignore_roles else ()
+        )
```

Comment:
maybe use a list for `args` instead, and append stuff when conditions are satisfied